src/site/xdoc/google_style.xml

```diff
@@ -2031,7 +2029,7 @@
                         src="images/ban_red.png"
                         alt="" />
                   </span>
-                  Proper validation require parsing other files, sources
+                  Proper validation requires parsing other files, sources
                   are not always available.
```

Comment:
This message should be the same as whatever we come up with for the message above